### DISCUSSION TITLE
post/ifkakao2024-dr-pym-project/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#108] [Announcements]
# post/ifkakao2024-dr-pym-project/

if(kakaoAI)2024에서 발표된 "Kubernetes 지능형 리소스 최적화"에 대해서 좀 더 상세히 공유하고자 합니다.

https://tech.kakaopay.com/post/ifkakao2024-dr-pym-project/

**Comment by smilejj91** [comment]
좋은 글 공유 감사드립니다. :)
